### DISCUSSION TITLE
使用windows版本在本地部署，局域网不能访问

### DISCUSSION THREAD

**Discussion by killua3000** [AlistGo/alist#796] [Q&A]
如题，根据文档中教程，在windows系统下打开alist-xxxx.exe，在本机浏览器中输入IP:5244能正常访问，局域网中比如手机设备无法通过IP:5244进行访问，在文档中似乎也没有相关描述，不知是否正常？如果需要手机浏览，还应如何配置？求作者指导一下，感谢！

**Comment by xhofe** [comment]
网络问题自己解决

**Reply by xhofe** [reply]
我只能说Windows也有防火墙 具体自己百度。我也不常使用Windows。

**Reply by hengzi-hub** [reply]
> 我只能说Windows也有防火墙 具体自己百度。我也不常使用Windows。

大佬你好，我的MacOS也遇到这个问题了，这个得怎么解决啊。。

**Reply by myfreewiki** [reply]
ddnsto.com内网穿透
